match/lib/match/utils.rb

```diff
@@ -1,6 +1,17 @@
 module Match
   class Utils
     def self.import(item_path, keychain)
+      keychain_path = self.keychain_path(keychain)
```

Comment:
I refactored the code for finding the path to a keychain in its own method. That method is needed in generator.rb to find the keychain path for cert.